kafka/kafka-3.6.0/image.yaml

```diff
@@ -2,12 +2,12 @@ schema_version: 1
 
 name: amq-streams/kafka-36-rhel8
 description: "AMQ Streams image for running Apache Kafka, Zookeeper, Kafka Connect, Mirror Maker and Cruise Control"
-version: "2.6.0"
-from: registry.redhat.io/ubi8/ubi-minimal
+version: "2.7.0"
+from: registry.redhat.io/rhel8-6-els/rhel
 
 labels:
   - name: "com.redhat.component"
-    value: "amqstreams-kafka-36-container"
+    value: "amqstreams-kafka-35-container"
```

Comment:
Is this change correct? Shouldn't it be -> 37 rather than -> 35?

Reply:
Fixed